src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -20,6 +20,7 @@
 // that server and then redirect incoming requests to it over standard HTTP on
 // the loopback network interface.
 
+#include <iostream>
```

Comment:
Is this actually used, or was it for some debugging? FWIW, I recommend `KJ_DBG` for debug prints.

Reply:
Yeah, it was from debugging. Removed.